runtimes/standalone/src/lib.rs

```diff
@@ -118,7 +119,10 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	authoring_version: 4,
 	spec_version: 10740,
 	impl_version: 0,
+	#[cfg(not(feature = "disable-runtime-api"))]
 	apis: RUNTIME_API_VERSIONS,
+	#[cfg(feature = "disable-runtime-api")]
+	apis: sp_version::create_apis_vec![[]],
```

Comment:
interesting. Do you know why this was added?

Reply:
I honestly did not think beyond "yeah maybe we want to be able to disable the runtime API for some testing at some point" since this is part of all relaychain runtimes. However, that has been the case for a long time which I overlooked. So I am perfectly fine with reverting these changes.

Reply:
It's standalone so keep it in. We might find a use for it someday. 😂

Reply:
Its also in the other runtimes 🙈

Reply:
In that case i wouldn't want something in our spiritnet runtime if we don't have a use case for it.